src/client/dom-processor/client-dom-adapter.js

```diff
@@ -110,4 +111,8 @@ export default class ClientDomAdapter extends BaseDomAdapter {
     sameOriginCheck (location, checkedUrl) {
         return sameOriginCheck(location, checkedUrl);
     }
+
+    isExistingTarget (target) {
+        return findByName(target);
```

Comment:
`findByName` returns window. `isExistingTarget` should returns boolean.
So, rewrite as
```js
 isExistingTarget (target) {
        return !!findByName(target);
}
```